pages/login.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div class="login-page">
+    <v-btn
+      rounded
+      fab
+      small
+      icon
+      :ripple="false"
+      class="nav-back-btn"
+      @click="to"
+    >
+      <v-icon>mdi-arrow-left-circle-outline</v-icon>
+    </v-btn>
+    <div class="login-form">
+      <v-tabs v-model="currentTab">
+        <v-tab :ripple="false" v-for="tab in tabs" :key="tab">{{ tab }}</v-tab>
+      </v-tabs>
+
+      <v-tabs-items v-model="currentTab">
+        <v-tab-item>
+          <form class="login-form" @submit.prevent="submitHandler">
```

Comment:
we have the same form repeated three times. 

I have two suggestions: 
1) Use one form for all tabs and use the name of the tab as the identifier of where the form comes from
2) Make a form component and display it in tabs

I like the first option better. What do you think?

Reply:
I agree with the first option. even if we have some different fields in the future, we will be able to add them according to the tab we are

Reply:
first option is great if we keep same layout and same input order, lets see on next steps how we can improve this